internal/controllers/submariner/broker_controller_test.go

```diff
@@ -85,6 +114,55 @@ var _ = Describe("Broker controller tests", func() {
 		Expect(t.GeneralClient.Get(ctx, client.ObjectKey{Name: "serviceimports.multicluster.x-k8s.io"}, crd)).To(Succeed())
 	})
 
+	It("should create the CA certificate secret", func(ctx SpecContext) {
+		t.AssertReconcileSuccess(ctx)
+
+		caSecretClient := fakeDynClient.Resource(corev1.SchemeGroupVersion.WithResource("secrets")).Namespace(submarinerNamespace)
+
+		Eventually(func(g Gomega) {
+			caSecretObj, err := caSecretClient.Get(ctx, "submariner-ca", metav1.GetOptions{})
+			g.Expect(err).NotTo(HaveOccurred())
+
+			caSecret := resource.MustFromUnstructured(caSecretObj, &corev1.Secret{})
+
+			g.Expect(caSecret.Data).To(HaveKey("ca.crt"))
+			g.Expect(caSecret.Data).To(HaveKey("ca.key"))
+		}).Should(Succeed())
+	})
+
+	When("the Broker resource is deleted", func() {
+		var signingRequestor certificate.SigningRequestor
```

Comment:
Newlines in between sections improve readability.



```suggestion
		var signingRequestor certificate.SigningRequestor
		
```